src/Microsoft.Sbom.Api/Output/ManifestToolJsonSerializer.cs

```diff
@@ -59,10 +59,7 @@ public void Write(JsonDocument jsonDocument)
             return;
         }
 
-        using (jsonDocument)
-        {
-            jsonDocument.WriteTo(jsonWriter);
-        }
+        jsonDocument.WriteTo(jsonWriter);
```

Comment:
Pretty sure there are some comments about this that we'll want to update before merging

Reply:
Update the comments here and in the Interface that this class implements, is there anywhere else you were thinking of?